source/_integrations/nest.markdown

```diff
@@ -45,36 +45,94 @@ You are in control of the information and capabilities exposed to Home Assistant
+1. Search for **Smart Device management** and enable the API.
+    <img src='/images/integrations/nest/enable_sdm_api.png'>
 
-Follow all of the instructions in [Device Access: Quick Start Guide](https://developers.google.com/nest/device-access/get-started) carefully as it is easy to make a configuration mistake that is difficult to debug. It is recommended to exercise the entire guide, including the command to test out the API, to make sure that it is working before configuring Home Assistant.
+## Pub/Sub subscriber setup
 
-When you get to the steps about configuring events make sure to follow guide under [Events](https://developers.google.com/nest/device-access/api/events) that configures the [Pub/Sub subscription](https://console.cloud.google.com/cloudpubsub/subscription/list) from the Google Cloud console. Make sure to use the *topic name* from the device access console and a unique subscription ID in the cloud console. Note the message retention is how long messages will queue while offline, so keep that short (e.g., under an hour) to avoid a potentially large backlog of updates.
+The next phase is to enable the Pub/Sub API by creating a subscription that can keep Home Assistant informed of events or device changes in near real time. See [Device Access: Events](https://developers.google.com/nest/device-access/api/events) for the full detailed instructions.
+
+1. Visit [Enable the Cloud Pub/Sub API](https://console.developers.google.com/apis/library/pubsub.googleapis.com) in the Cloud Console and click **Enable**
+
+1. Go to the [Google Cloud Platform: Pub/Sub:
+  Subscriptions](https://console.cloud.google.com/cloudpubsub/subscription/list) page and click **Create Subscription**.
+
+1. You will need to pick a *Subscription ID*
+
+    <img src='/images/integrations/nest/create_subscription.png'>
+
+1. The *Topic name* should match the topic name in your project in the [Device Access Console](https://console.nest.google.com/device-access/) and typically looks like `projects/sdm-prod/topics/EXAMPLE`.
+
+    <img src='/images/integrations/nest/device_access_pubsub_topic.png'>
+
+1. Select **Pull** as the *Delivery Type*.
+
+1. Lower the message retention duration to be something short (e.g. 10 minutes or under an hour) to avoid a large backlog of updates when Home Assistant is turned off.
+
+1. Leave the rest of the defaults and click **Create**.
+
+1. Once created, copy the *Subscription name* which you will want to hold on to as you `subscriber_id` for configuring Home Assistant. This typically looks like `projects/MY-CLOUD-ID/subscriptions/EXAMPLE`.  Don't confuse *Subscription name* with *Topic name* since they look similar.
```

Comment:
```suggestion
1. Once created, copy the *Subscription name* which you will want to hold on to as your `subscriber_id` for configuring Home Assistant. This typically looks like `projects/MY-CLOUD-ID/subscriptions/EXAMPLE`. Don't confuse *Subscription name* with *Topic name* since they look similar.
```